acqa-repo1-aws-tf12-part1.tf

```diff
@@ -325,21 +340,21 @@ resource "aws_kms_key" "acqa-test-kmskey1" {
   description             = "acqa-test-kmskey1"
   deletion_window_in_days = 30
   tags = {
-    Name = format("%s-kmskey1", var.acqaPrefix)
+    Name         = format("%s-kmskey1", var.acqaPrefix)
     ACQAResource = "true"
-    Owner = "ACQA"
+    Owner        = "ACQA"
   }
 }
 
 # ebs volume
 resource "aws_ebs_volume" "acqa-test-ebsvolume1" {
   availability_zone = "ca-central-1a"
   size              = 25
-  encrypted         = false
+  encrypted         = true
   tags = {
-    Name = format("%s-ebsvolume1", var.acqaPrefix)
+    Name         = format("%s-ebsvolume1", var.acqaPrefix)
     ACQAResource = "true"
-    Owner = "ACQA"
+    Owner        = "ACQA"
   }
 }
```

Comment:
**Accurics remediation**
 
**[HIGH]: Ensure IMDv1 is disabled for AWS EC2 instances**
 
**Vulnerability**
AWS EC2 instances that have IMDv1 enabled are vulnerable to server side request forgery (SSRF) attacks.
 
**Remediation**
IMDv1 is vulnerable to SSRF attack. Thus, EC2 Instance should have metadata_options block with either http_endpoint attribute set to disabled or ensure http_tokens attribute is set to required. This ensures IMDv2 is being used.

source: accurics-